public/backend/js/admins.js

```diff
@@ -1,7 +1,30 @@
 $(document).ready(function(){
+  //datatables
   $('#list_users').DataTable();
   $('#list_books').DataTable();
   $('#list_categories').DataTable();
+
+  //Confirm delete
+  $('#confirmDelete').on('show.bs.modal', function (e) {
+  	  // set message
+      $message = $(e.relatedTarget).attr('data-message');
+      $(this).find('.modal-body p').text($message);
```

Comment:
[rank B] we can select element directly, right? Why do you have to find it here? 
Please check other too.